assets/_scss/elements/_typography.scss

```diff
@@ -17,12 +17,22 @@ p {
 }
 
 a {
-  color: $color-base-link;
-  &:hover, &:active, &:visited {
-    color: $color-base-link;
+  border-bottom: 1px dotted;
+  &:hover {
+    border-bottom: 1px solid;
   }
 }
 
+.usa-link {
+  border-bottom: none;
+  font-family: $font-sans; 
+  font-weight: 700;
```

Comment:
You might want to make font weights variables. If a user changes to a different fonts, they may need to reset what the various weights are according to their font.